avalon/io.py

```diff
@@ -199,7 +199,7 @@ def _from_environment():
         ) if os.getenv(item[0], item[1]) is not None
     }
 
-    session["schema"] = "avalon-core:session-1.0"
+    session["schema"] = "avalon-core:session-2.0"
```

Comment:
Just to confirm, can someone confirm this is backwards compatible? We aren't asserting any particular version anywhere I think?